frontend/src/components/explorer/MapViewer.vue

```diff
@@ -89,6 +90,7 @@ import { debounce } from 'vue-debounce';
 import DataOverlay from '@/components/explorer/mapViewer/DataOverlay.vue';
 import ErrorPanel from '@/components/shared/ErrorPanel';
```

Comment:
Not related to the pull request, but I'm curious what is the difference between importing with `.vue` and without.
e.g. 
import DataOverlay from '@/components/explorer/mapViewer/DataOverlay.vue'; 
  vs 
import ErrorPanel from '@/components/shared/ErrorPanel';

Reply:
Ah interesting. I'm not sure about it but apparently it seems to work with both, I imagine it also depends on the JavaScript syntax configuration one sets for the project. We should probably decide on a consensus about this.

Reply:
Good idea, let's take this separately.